src/multisignature.h

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2017-2018 The PIVX developers
+// Distributed under the MIT/X11 software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef PIVX_MULTISIGNATURE_H
+#define PIVX_MULTISIGNATURE_H
+
+
+#include <script/script.h>
+#include <utility>
```

Comment:
improper include ordering.

Also, `<script/script.h>` should use quote styling instead of brace styling for consistency.